### ISSUE TITLE
Feature! Gui for easy linking

### ISSUE THREAD

**Issue by TravisC671** [denverquane/automuteus#174]
it would be great if you had the discord bot & the application communicate in a way so that the host could see the users in the call, and the players in the lobby. I guess if you were to do this you could have it grab the names in the VC, and then you would have a simple Gui that would show available colors and the names, then the host would drag and drop the people to their names. this would be helpful for changing on the fly, as well as playing with my friends on mobile. Apple phones do not allow apps to run in the background. so when you swap to discord you are disconnected, making it a pain.

in conclusion, this is what I would think would be a great addition to this already amazing bot!
- Gui that is a quicker way of linking discord names to colors than doing .au link @blank

Thank you so much for this bot! It is the thing I have been needing for a long time!

**Comment by jasikpark** [comment]
Is this completed via the emoji reactions now? Or is that not accurate?

**Comment by JoshuaUndercover** [comment]
@TravisC671 If you set your name in game to your Discord name it auto links :) But as @jasikpark said, you can just react to the message and it'll set your color, no need to type out .au link. Everyone should be able to handle their own color :)
